subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -329,7 +329,8 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
     }
 
     SmokeTestGradleRunner scanRunner(String... args) {
-        runner("build", "-Dscan.dump", *args).forwardOutput()
+        // Run with --build-cache to test also build-cache events
+        runner("build", "-Dscan.dump", "--build-cache", *args).forwardOutput()
```

Comment:
❓ Do you need to configure a separate directory for the local build cache so the shared one isn't polluted?

Reply:
Fixed